## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 489874`
Commit: `3e2365433ee72bad73717d164b7fd7749082bce3`

---
### `x86_64-linux`
<details>
  <summary>:x: 25 packages failed to build:</summary>
  <ul>
    <li>envoy</li>
    <li>pomerium</li>
    <li>proxysql</li>
    <li>python314Packages.blackjax</li>
    <li>python314Packages.blackjax.dist</li>
    <li>python314Packages.cvxpy</li>
    <li>python314Packages.cvxpy.dist</li>
    <li>python314Packages.jaxopt</li>
    <li>python314Packages.jaxopt.dist</li>
    <li>python314Packages.pepit</li>
    <li>python314Packages.pepit.dist</li>
    <li>python314Packages.qutip</li>
    <li>python314Packages.qutip.dist</li>
    <li>sbclPackages.buildnode</li>
    <li>sbclPackages.buildnode-xhtml</li>
    <li>sbclPackages.cl-ana</li>
    <li>sbclPackages.cl-csv</li>
    <li>sbclPackages.cl-interpol</li>
    <li>sbclPackages.css-selectors</li>
    <li>sbclPackages.css-selectors-simple-tree</li>
    <li>sbclPackages.css-selectors-stp</li>
    <li>sbclPackages.data-table</li>
    <li>sbclPackages.gsll</li>
    <li>sbclPackages.lisp-unit2</li>
    <li>sbclPackages.mgl</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 538 packages built:</summary>
  <ul>
    <li>OVMF</li>
    <li>OVMF-cloud-hypervisor</li>
    <li>OVMF-cloud-hypervisor.fd</li>
    <li>OVMF.fd</li>
    <li>OVMFFull</li>
    <li>OVMFFull.fd</li>
    <li>_2ship2harkinian</li>
    <li>adoptopenjdk-icedtea-web</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aitrack</li>
    <li>alpaca</li>
    <li>appvm</li>
    <li>babeldoc</li>
    <li>babeldoc.dist</li>
    <li>box2d</li>
    <li>calibre</li>
    <li>calibre-no-speech</li>
    <li>cemu</li>
    <li>collectd</li>
    <li>comaps</li>
    <li>convertx</li>
    <li>crewai (python313Packages.crewai)</li>
    <li>crewai.dist (python313Packages.crewai.dist)</li>
    <li>crow-translate</li>
    <li>dartsim</li>
    <li>dawarich</li>
    <li>deface</li>
    <li>deface.dist</li>
    <li>diffoscope</li>
    <li>diffoscope.dist</li>
    <li>diffoscope.man</li>
    <li>edgetx</li>
    <li>edk2</li>
    <li>edk2-uefi-shell</li>
    <li>envision</li>
    <li>etcd (etcd_3_6)</li>
    <li>exo</li>
    <li>exo.dist</li>
    <li>fdroidserver</li>
    <li>fdroidserver.dist</li>
    <li>field-monitor</li>
    <li>firefox-beta-unwrapped</li>
    <li>firefox-beta-unwrapped.debug</li>
    <li>firefox-beta-unwrapped.symbols</li>
    <li>firefox-devedition-unwrapped</li>
    <li>firefox-devedition-unwrapped.debug</li>
    <li>firefox-devedition-unwrapped.symbols</li>
    <li>firefox-unwrapped</li>
    <li>firefox-unwrapped.debug</li>
    <li>firefox-unwrapped.symbols</li>
    <li>firefoxpwa</li>
    <li>frigate</li>
    <li>gnome-boxes</li>
    <li>goldboot</li>
    <li>hieroglyphic</li>
    <li>hns</li>
    <li>hns.dist</li>
    <li>ieda</li>
    <li>imgui</li>
    <li>imgui-file-dialog</li>
    <li>imgui-file-dialog.dev</li>
    <li>imgui.lib</li>
    <li>immich-machine-learning</li>
    <li>immich-machine-learning.dist</li>
    <li>imnodes</li>
    <li>imnodes.dev</li>
    <li>implot</li>
    <li>kikoplay</li>
    <li>kraft</li>
    <li>lacus</li>
    <li>lacus.dist</li>
    <li>libguestfs (python313Packages.guestfs)</li>
    <li>libguestfs.guestfsd (python313Packages.guestfs.guestfsd)</li>
    <li>libnvidia-container</li>
    <li>librenms</li>
    <li>librewolf-unwrapped</li>
    <li>librewolf-unwrapped.debug</li>
    <li>libvirt</li>
    <li>libvirt-dbus</li>
    <li>libvirt-dbus.man</li>
    <li>libvirt-glib</li>
    <li>libvirt-glib.dev</li>
    <li>libvirt-glib.devdoc</li>
    <li>libvmi</li>
    <li>libvmi.dev</li>
    <li>libvmi.lib</li>
    <li>livecaptions</li>
    <li>maa-assistant-arknights</li>
    <li>maa-cli</li>
    <li>magika (python313Packages.magika)</li>
    <li>magika-cli</li>
    <li>magika.dist (python313Packages.magika.dist)</li>
    <li>mastodon</li>
    <li>mgmt</li>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>minikube</li>
    <li>monado</li>
    <li>newelle</li>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>ocamlPackages.gettext-camomile</li>
    <li>ocamlPackages.gettext-stub</li>
    <li>ocamlPackages.ocaml_gettext</li>
    <li>ocamlPackages.ocaml_libvirt</li>
    <li>ocamlPackages.owl</li>
    <li>ocamlPackages.owl-base</li>
    <li>ocamlPackages_latest.gettext-camomile</li>
    <li>ocamlPackages_latest.gettext-stub</li>
    <li>ocamlPackages_latest.ocaml_gettext</li>
    <li>ocamlPackages_latest.ocaml_libvirt</li>
    <li>ocamlPackages_latest.owl</li>
    <li>ocamlPackages_latest.owl-base</li>
    <li>onnxruntime</li>
    <li>onnxruntime.debug</li>
    <li>onnxruntime.dev</li>
    <li>onnxruntime.dist</li>
    <li>open-webui</li>
    <li>open-webui.dist</li>
    <li>openseeface</li>
    <li>opentrack</li>
    <li>owocr</li>
    <li>owocr.dist</li>
    <li>perlPackages.SysVirt (perl5Packages.SysVirt)</li>
    <li>perlPackages.SysVirt.devdoc (perl5Packages.SysVirt.devdoc)</li>
    <li>piper-phonemize</li>
    <li>piper-tts</li>
    <li>piper-tts.dist</li>
    <li>podman-bootc</li>
    <li>pods</li>
    <li>prometheus-libvirt-exporter</li>
    <li>prowlarr</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ale-py.dist</li>
    <li>python313Packages.arviz</li>
    <li>python313Packages.arviz.dist</li>
    <li>python313Packages.bambi</li>
    <li>python313Packages.bambi.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.brax.dist</li>
    <li>python313Packages.chromadb</li>
    <li>python313Packages.chromadb.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.clu</li>
    <li>python313Packages.clu.dist</li>
    <li>python313Packages.corner</li>
    <li>python313Packages.corner.dist</li>
    <li>python313Packages.cvxpy</li>
    <li>python313Packages.cvxpy.dist</li>
    <li>python313Packages.dalle-mini</li>
    <li>python313Packages.dalle-mini.dist</li>
    <li>python313Packages.dartsim</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.deepface.dist</li>
    <li>python313Packages.distrax</li>
    <li>python313Packages.distrax.dist</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.dm-sonnet.dist</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.edward.dist</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.evosax.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.fastembed</li>
    <li>python313Packages.fastembed.dist</li>
    <li>python313Packages.faster-whisper</li>
    <li>python313Packages.faster-whisper.dist</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flax.dist</li>
    <li>python313Packages.flowjax</li>
    <li>python313Packages.flowjax.dist</li>
    <li>python313Packages.gliner</li>
    <li>python313Packages.gliner.dist</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.gymnasium.dist</li>
    <li>python313Packages.helium</li>
    <li>python313Packages.helium.dist</li>
    <li>python313Packages.insightface</li>
    <li>python313Packages.insightface.dist</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.keras.dist</li>
    <li>python313Packages.kokoro</li>
    <li>python313Packages.kokoro.dist</li>
    <li>python313Packages.lacuscore</li>
    <li>python313Packages.lacuscore.dist</li>
    <li>python313Packages.langchain-chroma</li>
    <li>python313Packages.langchain-chroma.dist</li>
    <li>python313Packages.langchain-huggingface</li>
    <li>python313Packages.langchain-huggingface.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.libvirt</li>
    <li>python313Packages.libvirt.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.llm-sentence-transformers</li>
    <li>python313Packages.llm-sentence-transformers.dist</li>
    <li>python313Packages.manifest-ml</li>
    <li>python313Packages.manifest-ml.dist</li>
    <li>python313Packages.markitdown</li>
    <li>python313Packages.markitdown.dist</li>
    <li>python313Packages.meeko</li>
    <li>python313Packages.meeko.dist</li>
    <li>python313Packages.mhcflurry</li>
    <li>python313Packages.mhcflurry.dist</li>
    <li>python313Packages.mim-solvers</li>
    <li>python313Packages.mmcv</li>
    <li>python313Packages.mmcv.dist</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mtcnn.dist</li>
    <li>python313Packages.numpyro</li>
    <li>python313Packages.numpyro.dist</li>
    <li>python313Packages.nutpie</li>
    <li>python313Packages.nutpie.dist</li>
    <li>python313Packages.onnx-ir</li>
    <li>python313Packages.onnx-ir.dist</li>
    <li>python313Packages.onnxruntime</li>
    <li>python313Packages.onnxruntime.dist</li>
    <li>python313Packages.onnxscript</li>
    <li>python313Packages.onnxscript.dist</li>
    <li>python313Packages.osqp</li>
    <li>python313Packages.osqp.dist</li>
    <li>python313Packages.pepit</li>
    <li>python313Packages.pepit.dist</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pettingzoo.dist</li>
    <li>python313Packages.piper-phonemize</li>
    <li>python313Packages.piper-phonemize.dist</li>
    <li>python313Packages.pymc</li>
    <li>python313Packages.pymc.dist</li>
    <li>python313Packages.pypasser</li>
    <li>python313Packages.pypasser.dist</li>
    <li>python313Packages.pytensor</li>
    <li>python313Packages.pytensor.dist</li>
    <li>python313Packages.qpsolvers</li>
    <li>python313Packages.qpsolvers.dist</li>
    <li>python313Packages.qutip</li>
    <li>python313Packages.qutip.dist</li>
    <li>python313Packages.rapidocr</li>
    <li>python313Packages.rapidocr-onnxruntime</li>
    <li>python313Packages.rapidocr-onnxruntime.dist</li>
    <li>python313Packages.rapidocr.dist</li>
    <li>python313Packages.rdkit</li>
    <li>python313Packages.rembg</li>
    <li>python313Packages.rembg.dist</li>
    <li>python313Packages.retinaface</li>
    <li>python313Packages.retinaface.dist</li>
    <li>python313Packages.rlax</li>
    <li>python313Packages.rlax.dist</li>
    <li>python313Packages.scikit-survival</li>
    <li>python313Packages.scikit-survival.dist</li>
    <li>python313Packages.sentence-transformers</li>
    <li>python313Packages.sentence-transformers.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.speechrecognition</li>
    <li>python313Packages.speechrecognition.dist</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.stable-baselines3.dist</li>
    <li>python313Packages.tami4edgeapi</li>
    <li>python313Packages.tami4edgeapi.dist</li>
    <li>python313Packages.tensorflow-probability</li>
    <li>python313Packages.tensorflow-probability.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.tf2onnx</li>
    <li>python313Packages.tf2onnx.dist</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.tianshou.dist</li>
    <li>python313Packages.tinygrad</li>
    <li>python313Packages.tinygrad.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.txtai</li>
    <li>python313Packages.txtai.dist</li>
    <li>python313Packages.tyro</li>
    <li>python313Packages.tyro.dist</li>
    <li>ultralytics (python313Packages.ultralytics)</li>
    <li>ultralytics.dist (python313Packages.ultralytics.dist)</li>
    <li>python313Packages.unsloth</li>
    <li>python313Packages.unsloth-zoo</li>
    <li>python313Packages.unsloth-zoo.dist</li>
    <li>python313Packages.unsloth.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vqgan-jax</li>
    <li>python313Packages.vqgan-jax.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.waymax.dist</li>
    <li>xen (python313Packages.xen)</li>
    <li>xen.boot (python313Packages.xen.boot)</li>
    <li>xen.dev (python313Packages.xen.dev)</li>
    <li>xen.doc (python313Packages.xen.doc)</li>
    <li>xen.man (python313Packages.xen.man)</li>
    <li>python314Packages.dartsim</li>
    <li>python314Packages.fastembed</li>
    <li>python314Packages.fastembed.dist</li>
    <li>python314Packages.faster-whisper</li>
    <li>python314Packages.faster-whisper.dist</li>
    <li>python314Packages.gliner</li>
    <li>python314Packages.gliner.dist</li>
    <li>python314Packages.guestfs</li>
    <li>python314Packages.guestfs.guestfsd</li>
    <li>python314Packages.helium</li>
    <li>python314Packages.helium.dist</li>
    <li>python314Packages.insightface</li>
    <li>python314Packages.insightface.dist</li>
    <li>python314Packages.k-diffusion</li>
    <li>python314Packages.k-diffusion.dist</li>
    <li>python314Packages.lacuscore</li>
    <li>python314Packages.lacuscore.dist</li>
    <li>python314Packages.langchain-huggingface</li>
    <li>python314Packages.langchain-huggingface.dist</li>
    <li>python314Packages.libvirt</li>
    <li>python314Packages.libvirt.dist</li>
    <li>python314Packages.llm-sentence-transformers</li>
    <li>python314Packages.llm-sentence-transformers.dist</li>
    <li>python314Packages.magika</li>
    <li>python314Packages.magika.dist</li>
    <li>python314Packages.manifest-ml</li>
    <li>python314Packages.manifest-ml.dist</li>
    <li>python314Packages.markitdown</li>
    <li>python314Packages.markitdown.dist</li>
    <li>python314Packages.meeko</li>
    <li>python314Packages.meeko.dist</li>
    <li>python314Packages.mim-solvers</li>
    <li>python314Packages.onnx-ir</li>
    <li>python314Packages.onnx-ir.dist</li>
    <li>python314Packages.onnxruntime</li>
    <li>python314Packages.onnxruntime.dist</li>
    <li>python314Packages.onnxscript</li>
    <li>python314Packages.onnxscript.dist</li>
    <li>python314Packages.osqp</li>
    <li>python314Packages.osqp.dist</li>
    <li>python314Packages.piper-phonemize</li>
    <li>python314Packages.piper-phonemize.dist</li>
    <li>python314Packages.pypasser</li>
    <li>python314Packages.pypasser.dist</li>
    <li>python314Packages.qpsolvers</li>
    <li>python314Packages.qpsolvers.dist</li>
    <li>python314Packages.rapidocr</li>
    <li>python314Packages.rapidocr-onnxruntime</li>
    <li>python314Packages.rapidocr-onnxruntime.dist</li>
    <li>python314Packages.rapidocr.dist</li>
    <li>python314Packages.rdkit</li>
    <li>python314Packages.rembg</li>
    <li>python314Packages.rembg.dist</li>
    <li>python314Packages.scikit-survival</li>
    <li>python314Packages.scikit-survival.dist</li>
    <li>python314Packages.sentence-transformers</li>
    <li>python314Packages.sentence-transformers.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.speechrecognition</li>
    <li>python314Packages.speechrecognition.dist</li>
    <li>python314Packages.tami4edgeapi</li>
    <li>python314Packages.tami4edgeapi.dist</li>
    <li>python314Packages.tinygrad</li>
    <li>python314Packages.tinygrad.dist</li>
    <li>python314Packages.txtai</li>
    <li>python314Packages.txtai.dist</li>
    <li>python314Packages.ultralytics</li>
    <li>python314Packages.ultralytics.dist</li>
    <li>python314Packages.unstructured-inference</li>
    <li>python314Packages.unstructured-inference.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
    <li>python314Packages.xen</li>
    <li>python314Packages.xen.boot</li>
    <li>python314Packages.xen.dev</li>
    <li>python314Packages.xen.doc</li>
    <li>python314Packages.xen.man</li>
    <li>qemu_xen</li>
    <li>qemu_xen.debug</li>
    <li>qemu_xen.doc</li>
    <li>qemu_xen.ga</li>
    <li>quickemu</li>
    <li>quickgui</li>
    <li>quickgui.debug</li>
    <li>quickgui.pubcache</li>
    <li>radarr</li>
    <li>rapidraw</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>rubyPackages.ruby-libvirt (rubyPackages_3_3.ruby-libvirt)</li>
    <li>rubyPackages_3_4.ruby-libvirt</li>
    <li>rubyPackages_4_0.ruby-libvirt</li>
    <li>sbclPackages.cephes</li>
    <li>sbclPackages.cl-cairo2-xlib</li>
    <li>sbclPackages.cl-change-case</li>
    <li>sbclPackages.cl-freetype2</li>
    <li>sbclPackages.cl-fuse</li>
    <li>sbclPackages.cl-fuse-meta-fs</li>
    <li>sbclPackages.cl-liballegro-nuklear</li>
    <li>sbclPackages.cl-ppcre-unicode</li>
    <li>sbclPackages.cl-sat_dot_glucose</li>
    <li>sbclPackages.cl-unicode</li>
    <li>sbclPackages.clack</li>
    <li>sbclPackages.hu_dot_dwim_dot_def_plus_swank</li>
    <li>sbclPackages.hu_dot_dwim_dot_stefil_plus_hu_dot_dwim_dot_def_plus_swank</li>
    <li>sbclPackages.hu_dot_dwim_dot_stefil_plus_swank</li>
    <li>sbclPackages.kons-9</li>
    <li>sbclPackages.magicl</li>
    <li>sbclPackages.polyclot</li>
    <li>sbclPackages.qlot-cli</li>
    <li>sbclPackages.query-fs</li>
    <li>sbclPackages.stefil</li>
    <li>sbclPackages.str</li>
    <li>sbclPackages.swank</li>
    <li>seagoat</li>
    <li>seagoat.dist</li>
    <li>shipwright</li>
    <li>sitespeed-io</li>
    <li>sonarr</li>
    <li>sony-headphones-client</li>
    <li>spaghettikart</li>
    <li>terminusdb</li>
    <li>tests.fetchNextcloudApp.simple-sha256</li>
    <li>tests.fetchNextcloudApp.simple-sha512</li>
    <li>tests.home-assistant-component-tests.tami4</li>
    <li>thunderbird-unwrapped (thunderbird-latest-unwrapped, thunderbirdPackages.thunderbird, thunderbirdPackages.thunderbird-latest)</li>
    <li>thunderbird-unwrapped.debug (thunderbird-latest-unwrapped.debug, thunderbirdPackages.thunderbird-latest.debug, thunderbirdPackages.thunderbird.debug)</li>
    <li>thunderbird-unwrapped.symbols (thunderbird-latest-unwrapped.symbols, thunderbirdPackages.thunderbird-latest.symbols, thunderbirdPackages.thunderbird.symbols)</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>unbook</li>
    <li>unstructured-api</li>
    <li>vagrant</li>
    <li>vcmi</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
    <li>virt-manager</li>
    <li>virt-top</li>
    <li>virt-v2v</li>
    <li>virt-viewer</li>
    <li>virtnbdbackup</li>
    <li>virtnbdbackup.dist</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
    <li>wivrn</li>
    <li>wyoming-faster-whisper</li>
    <li>wyoming-faster-whisper.dist</li>
    <li>wyoming-piper</li>
    <li>wyoming-piper.dist</li>
    <li>xen-guest-agent</li>
    <li>xrdp</li>
    <li>zammad</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>proxysql</summary>
<pre>  CC       ../lib/curlx/libcurltool_la-timeval.lo
  CC       ../lib/curlx/libcurltool_la-version_win32.lo
  CC       ../lib/curlx/libcurltool_la-wait.lo
  CC       ../lib/curlx/libcurltool_la-warnless.lo
  CC       ../lib/curlx/libcurltool_la-winapi.lo
  CC       curl-tool_ca_embed.o
  CCLD     curlinfo
  CCLD     curl
  CCLD     libcurltool.la
make[3]: Leaving directory &#x27;/build/source/deps/curl/curl/src&#x27;
Making all in scripts
make[3]: Entering directory &#x27;/build/source/deps/curl/curl/scripts&#x27;
make[3]: Nothing to be done for &#x27;all&#x27;.
make[3]: Leaving directory &#x27;/build/source/deps/curl/curl/scripts&#x27;
make[3]: Entering directory &#x27;/build/source/deps/curl/curl&#x27;
make[3]: Nothing to be done for &#x27;all-am&#x27;.
make[3]: Leaving directory &#x27;/build/source/deps/curl/curl&#x27;
make[2]: Leaving directory &#x27;/build/source/deps/curl/curl&#x27;
make[1]: Leaving directory &#x27;/build/source/deps&#x27;
make: *** [Makefile:269: build_deps_default] Error 2</pre>
</details>
<details>
<summary>python314Packages.cvxpy</summary>
<pre>      used in CVXPY is Fortran (&#x27;F&#x27;) order. This default will change to match NumPy&#x27;s
      default order (&#x27;C&#x27;) in a future version of CVXPY.
      To suppress this warning, please specify the order explicitly.
      
    problem.solve(gp=True, solver=cp.GLPK_MI, enforce_dpp=True)

cvxpy/tests/test_valinvec2mixedint.py::test_default_argument
  /build/source/cvxpy/tests/test_valinvec2mixedint.py:409: FutureWarning: 
      You didn&#x27;t specify the order of the flatten expression. The default order
      used in CVXPY is Fortran (&#x27;F&#x27;) order. This default will change to match NumPy&#x27;s
      default order (&#x27;C&#x27;) in a future version of CVXPY.
      To suppress this warning, please specify the order explicitly.
      
    problem.solve(solver=cp.GLPK_MI)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED cvxpy/tests/test_problem.py::TestProblem::test_cp_node_count_warn - AssertionError: assert 1 == 0
 +  where 1 = len([&lt;WarningMessage {message : RuntimeWarning(&#x27;invalid value encountered in reduce&#x27;), category : &#x27;RuntimeWarning&#x27;, filena...d6phjqxcvn-python3.14-numpy-2.3.5/lib/python3.14/site-packages/numpy/_core/fromnumeric.py&#x27;, lineno : 86, line : None}&gt;])
=== 1 failed, 1917 passed, 466 skipped, 7 deselected, 442 warnings in 38.62s ===</pre>
</details>
<details>
<summary>sbclPackages.cl-interpol</summary>
<pre>;     (CL-INTERPOL::SIGNAL-READER-ERROR &quot;No character for char-code #x~X&quot; NUMBER)
; 
; note: deleting unreachable code


; wrote /build/source/util-tmp5GEXGEG5.fasl
; compilation finished in 0:00:00.005
; compiling file &quot;/nix/store/w8xkk5cg2spq45dskwn6fyyk5dfbxzhb-source/alias.lisp&quot; (written 01 JAN 1970 12:00:01 AM):

; wrote /build/source/alias-tmpAR3FSGEY.fasl
; compilation finished in 0:00:00.000
; compiling file &quot;/nix/store/w8xkk5cg2spq45dskwn6fyyk5dfbxzhb-source/read.lisp&quot; (written 01 JAN 1970 12:00:01 AM):

; wrote /build/source/read-tmpJAIDFZTC.fasl
; compilation finished in 0:00:00.057
; 
; compilation unit aborted
;   caught 1 fatal ERROR condition
;   printed 1 note
BUILD FAILED: Bug in readtable iterators or concurrent access?</pre>
</details>
</details>

--- a/pkgs/build-support/trivial-builders/default.nix
+++ b/pkgs/build-support/trivial-builders/default.nix
@@ -1053,10 +1053,7 @@ rec {
             if builtins.isPath finalAttrs.src then
               baseNameOf finalAttrs.src + "-patched"
             else if builtins.isAttrs finalAttrs.src && (finalAttrs.src ? name) then
-              let
-                srcName = builtins.parseDrvName finalAttrs.src.name;
-              in
-              "${srcName.name}-patched${lib.optionalString (srcName.version != "") "-${srcName.version}"}"
+              finalAttrs.src.name + "-patched"
             else
               throw "applyPatches: please supply a `name` argument because a default name can only be computed when the `src` is a path or is an attribute set with a `name` attribute."
           );
@@ -1082,7 +1079,7 @@ rec {
         passthru = extraPassthru // finalAttrs.src.passthru or { };
 
         # Carry (and merge) information from the underlying `src` if present.
-        meta = lib.optionalAttrs (src ? meta) (removeAttrs finalAttrs.src.meta) [ "position" ];
+        meta = lib.optionalAttrs (src ? meta) (removeAttrs finalAttrs.src.meta [ "position" ]);
       };
   };
 

